### PR TITLE
git:// URLs are not encrypted or authenticated

### PR DESCRIPTION
**Pull request by glyph**
This is a major security problem for any local Klein development.

### CODE REVIEW

**Review by graingert — APPROVED** [review]
You might prefer the GitHub archive `https://``tar.gz` as it's usually a bit faster

Also you might prefer using the pep440 `Twisted @ https://...` instead of the legacy `#egg=`

File: tox.ini [hunk 1]
```diff
@@ -307,7 +307,7 @@ commands =
 basepython = python3.8
 
 deps =
-    git+git://github.com/twisted/pydoctor@c74016b#egg=pydoctor
+    git+https://github.com/twisted/pydoctor@c74016b#egg=pydoctor
```
Comment (graingert) [inline tox.ini:310]:
```suggestion
    git+https://github.com/twisted/pydoctor@c74016b8266ff0c5cea64f9c035bf0a568832f0f#egg=pydoctor
```

Comment (graingert) [reply tox.ini:310]:
```suggestion
    pydoctor>=20.12.1
```

**Comment by glyph** [comment]
> You might prefer the GitHub archive `` https://``tar.gz `` as it's usually a bit faster
> 
> Also you might prefer using the pep440 `Twisted @ https://...` instead of the legacy `#egg=`

Happy to do these as follow-ups, this was just a quick fix to address the immediate bleeding :)

**Pull request merged**